incydr/cli/cmds/watchlists.py

```diff
@@ -22,15 +25,23 @@
 from incydr.cli.core import incompatible_with
 from incydr.cli.core import IncydrCommand
 from incydr.cli.core import IncydrGroup
+from incydr.cli.file_readers import FileOrString
 from incydr.cli.render import measure_renderable
 from incydr.cli.render import models_as_table
-from incydr.types import file_or_str_cls
 from incydr.utils import model_as_card
-from incydr.utils import read_dict_from_csv
 
 MAX_USER_DISPLAY_COUNT = 25
 
 
+class UserCSV(CSVModel):
+    user: str = Field(csv_aliases=["user", "user_id", "username", "id", "userId"])
+
+
+# TODO: what should this field be called?
```

Comment:
I'm not sure what the best name for the user field of the JSON models would be?

Reply:
I think it's reasonably safe to assume any json-lines input will be coming from the CLI itself, so we can probably only support `userId` and `username` aliases. Something like this should work:

```
class User(BaseModel):
    username: Optional[str]
    userId: Optional[str]

    @root_validator(pre=True)
    def _validate(cls, values):
        if not "username" in values and not "userId" in values:
            raise ValueError("A json key of 'username' or 'userId' is required")
        return values

    @property
    def user(self):
        return self.userId or self.username
```